project.clj

```diff
@@ -7,12 +7,21 @@
   :license {:name "Apache License, Version 2.0"
             :url "http://www.apache.org/licenses/LICENSE-2.0.html"}
 
-  :dependencies [[org.clojure/clojure "1.6.0"]
-                 [org.clojure/java.jdbc "0.3.2"]
-                 [org.postgresql/postgresql "9.3-1100-jdbc41"]
-                 [com.jolbox/bonecp "0.8.0.RELEASE" :exclusions [[org.slf4j/slf4j-api]]]
+  :dependencies [[com.jolbox/bonecp "0.8.0.RELEASE" :exclusions [[org.slf4j/slf4j-api]]]
+                 [com.zaxxer/HikariCP-java6 "2.3.7" :exclusions [[org.slf4j/slf4j-api]]]
+                 [metrics-clojure "2.5.1" :exclusions [[org.slf4j/slf4j-api]]]
```

Comment:
@kbarber would it be possible to just leave the metrics stuff out until we have a chance to consolidate things?  I'm worried that this is going to create a third implementation of metrics and that it'll just be that much harder to consolidate later.  Also, we're not even using metrics-clojure; we just use the Java library directly because the clojure wrapper didn't offer much and made some things harder than they are without it.